modules/standard/org/wave_counter_controller.py

```diff
@@ -16,11 +16,12 @@ def inject(self, registry):
         self.bot = registry.get_instance("bot")
         self.job_scheduler = registry.get_instance("job_scheduler")
 
-    @event(event_type=PublicChannelService.ORG_MSG_EVENT, description="Start wave counter when city is targeted by aliens")
+    @event(event_type=PublicChannelService.ORG_CHANNEL_MESSAGE_EVENT, description="Start wave counter when city is targeted by aliens")
```

Comment:
I don't think this change is correct.  I believe the messages for wave counters come as ORG_MSG_EVENT and not ORG_CHANNEL_MESSAGE_EVENT.  Can you verify?

Reply:
I'm 100% sure its ORG_CHANNEL_... an few ppl reportet it to me, that it was not working - with ORG_MSG. i've changed it to ORG_CHANNEL on my private bot an few months ago, since that moment it started spamming the waves.

The reason why its not ORG_MSG, is, at least of what i learned that far (could be wrong tho): ORG_CHANNEL is written in `[{ORG_NAME}] Your radar station is picking up alien activity in the area surrounding your city.`
and the org MSG's are prefixed with [ORG MSG]

chatlog WITH this change: 

`(19:54) [Dragons] Dragons: [Dragons] Viary turned the cloaking device in your city off.`
`(19:54) [Dragons] Dragons: [Dragons] Your radar station is picking up alien activity in the area surrounding your city.`
`(19:57) [Dragons] Dragons: [Dragons] Your city in Serenity Islands has been targeted by hostile forces.`
`(19:57) [Dragons] Dragons: [Dragons] Wave counter started.` 
(watched it in the priv channel)

Lets hope for some good loot, i guess? :P

Reply:
ok great